### ISSUE TITLE
Highlight function calls

### ISSUE THREAD

**Issue by dsl101** [Thom1729/Sublime-JS-Custom#87]
I've been using JavaScript Next for ages, and have really got used to it. But it's old, and now doesn't support things like arrow function parameter default values which are function calls amongst [other things](https://github.com/Benvie/JavaScriptNext.tmLanguage/issues/4). So I moved to JS Custom, but there is one thing that niggles me—I'm really used to function _calls_ highlighted as function names. Is it possible to fix with a local customisation? Or does it need a change to the syntax file itself? Compare JavaScript Next on left vs JS Custom on right. The `thing` function definition and parameters look great, but I'd like the calls to also be green...

![image](https://user-images.githubusercontent.com/900542/72890987-1be2bb80-3d0b-11ea-90f2-1f1962295ef5.png)

**Comment by Thom1729** [comment]
The function names in the calls are scoped `variable.function.js`, which should be colored by most schemes. Your color scheme looks like Monokai, but I've just checked and the version of Monokai bundled with Sublime does highlight the function calls. Are you using a third-party color scheme?

**Comment by dsl101** [comment]
I have both Monokai and Monokai (SL) installed—not sure if either came with JS Next. But both those schemes colour function calls in JS Next and don't in JS Custom.

I've removed JS Next, cleared the \Cache fodler and restarted ST, but still no coloring of function calls.

Then I installed Monokai++, and function call coloring is there. So I suspect I just have a random collection of old theme files which are out of date :). Sorry for the interruption! Still no idea why JS Next worked, unless it patched the color scheme when active... Anyway, problem solved.
